src/xpcc/architecture/driver/atomic/queue.hpp

```diff
@@ -67,6 +67,9 @@ namespace xpcc
 			ALWAYS_INLINE bool
 			isFull() const;
 			
+			ALWAYS_INLINE bool
+			isNotFull() const { return not isFull(); }
```

Comment:
Yeah, I don't want to actively rewrite existing code, because that'll bloat xpcc's git history even more. So if I stumble across it, I'll change it.

The thing I'm most annoyed about is actually checked in trailing whitespace.
My editor strips trailing white space, so every time I modify such a file, I have to carefully stage all my changes. Grrrr.